robotoff/app/api.py

```diff
@@ -1141,6 +1142,31 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         resp.media = response
 
 
+class UnansweredQuestionCollection:
+    def on_get(self, req: falcon.Request, resp: falcon.Response):
+        response: JSONType = {}
+        page: int = req.get_param_as_int("page", min_value=1, default=1)
+        count: int = req.get_param_as_int("count", min_value=1, default=25)
+        question_type: str = req.get_param("question_type")
```

Comment:
we can name the parameter type to be consistent (although avoiding the name for the variable).

```suggestion
        question_type: str = req.get_param("type")
```